modules/connectivity/ParticipantConnectionStatus.js

```diff
@@ -380,18 +424,53 @@ export default class ParticipantConnectionStatus {
             isConnActiveByJvb = true;
         }
 
-        const isConnectionActive
-            = isConnActiveByJvb
-                && (isVideoMuted || (isInLastN && !isVideoTrackFrozen));
+        let newState = ParticipantConnectionStatus.INACTIVE;
+
+        if (isConnActiveByJvb) {
+            if (isInLastN) {
```

Comment:
if the person is not in last N it will immediately be marked as INACTIVE ? I'd say it's not safe to assume connectivity issues this way. We may be showing "connectivity issues" for brief moments for good video if the lastN msg will not arrive in time.